src/main/java/org/folio/oaipmh/dao/PostgresClientFactory.java

```diff
@@ -36,6 +36,8 @@
 
   private static final int POOL_SIZE = 20;
 
+  private static boolean shouldResetPool = false;
```

Comment:
please add ToDo comment to remove this field in the future, with a description of why it present here,  as this field needed only for tests